kitchen/modules/smpl-treasury/src/lib.rs

```diff
@@ -1,14 +1,12 @@
-//! Batching Balance Transfers through an Automated Account
-//!
-//! # thinking about economic design while building modules
-//!
-//! 
-#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
Did you remove this line intentionally? Forgetting this in modules has caused me many "duplicate lang item ..." errors.

Reply:
I think it was accidental, I'll add it back -- thanks!